src/main/java/gov/nih/nci/icdc/IcdcEsFilter.java

```diff
@@ -913,4 +878,27 @@ private Map<String, Object> globalSearch(Map<String, Object> params) throws IOEx
         }
         return result;
     }
+
+    private static Map<String, Object> formatParams(Map<String, Object> params, String targetParam, String lowercaseParam) {
+        Map<String, Object> formattedParams = new HashMap<>();
+
+        params.forEach((key, value) -> {
+            if (key.equals(targetParam)) {
+                try {
+                    ArrayList<String> valuesList = (ArrayList<String>) value;
```

Comment:
I'm getting an unchecked type casting warning in my IDE because of this line - not sure how big of a deal that is. It goes away if I swap with `List<?> valuesList = (List<?>) value;`, but I guess I expect `valuesList` to always be an array of strings...

Reply:
I think this warning could be resolved by adding a catch block for a Class Cast Exception and then throwing an appropriate error message. I don't know think this exception would happen but it would not hurt to add this